src/js/themes/vanilla.js

```diff
@@ -260,7 +260,10 @@ export default deepFreeze({
   anchor: {
     textDecoration: 'none',
     fontWeight: 600,
-    color: css`${props => props.theme.global.colors.brand}`,
+    colors: {
```

Comment:
If we keep this named `color` instead of `colors`? That will align with https://github.com/grommet/grommet/pull/2151.

Reply:
no problem